### DISCUSSION TITLE
Release v1.0.1 / 5.55.1 (pre-release)

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1418] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes bugs introduced in the last pre-release build 

# Known Issues
-  printing does not work, will be fixed in the next release

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# Changelog

### Added
- added checkboxes to most major box options lists
- added SumatraPDF templates (by Dyras) [#1391](https://github.com/sandboxie-plus/Sandboxie/pull/1391)

### Changed
- rolled back change to "OpenClsid=..." handling
- made all major lists in the box options editable

### Fixed
- fixed issue with read only paths introduced in 1.0.0
- fixed BSOD issue introduced in the 1.0.0 build [#1389](https://github.com/sandboxie-plus/Sandboxie/issues/1389)
- fixed multiple BITS notifications while running sandboxed Chromium browsers (by isaak654) [ca320ec](https://github.com/sandboxie-plus/Sandboxie/commit/ca320ecc17180ff09a67bdefc524b30cf3540c08) [#1081](https://github.com/sandboxie-plus/Sandboxie/issues/1081)
- fixed executables selection for "Run Menu" entries (by isaak654) [#1379](https://github.com/sandboxie-plus/Sandboxie/issues/1379)
- fixed SetCursorPos and ClipCursor ignoring DPI awareness (by alvinhochun) [#1394](https://github.com/sandboxie-plus/Sandboxie/pull/1394)

### Removed
- removed Virtual Desktop Manager template (by isaak654) [d775807](https://github.com/sandboxie-plus/Sandboxie/commit/d7758071f6930539c4e1f236297b4cfa332346ad) [#1326](https://github.com/sandboxie-plus/Sandboxie/discussions/1326)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/'>Release v1.0.1 / 5.55.1</a>.</em>

**Comment by BilleBarrett** [comment]
I waited to start upgrading my clients to the latest Sandboxie and the problem started.
This is a major problem, my clients are calling me telling me that they can no longer print Gmails, or documents from inside Google chrome
I changed from Classic to Plus and the problem persists, 
please advise
Right now they have to use print to PDF and save the PDF and then print that. 
Everyone wants to revert back to the Sandboxie that this does not happen in
note: if I run outside of Sandboxie printing is no problem.
Ran a test, monitoring the Spool folder, I see the print data getting sent into the Print spooler, & it just sits there
I check the spool service is running, it is run the same outside of Sandboxie [clasic or Plus] it gets sent to the spool folder and then to the printer.
It seems something is stopping the handoff
For some reason my computer does not do this, I see the spool file get the print job & then it goes to the printer
Why is this not happening to my client's computers?
I also noted is supposed to be set in the INI file
OpenPrintSpooler=y
AllowSpoolerPrintToFile=y

**Reply by isaak654** [reply]
The printing issue will be fixed in the next version:
* https://github.com/sandboxie-plus/Sandboxie/issues/1397
* https://www.wilderssecurity.com/threads/cannot-print.442353/#post-3055618
